include/programs.h

```diff
@@ -97,6 +98,14 @@ class Program {
 	bool HelpRequested();
 
 	static void ResetLastWrittenChar(char c);
+<<<<<<< HEAD
+=======
+
+	void AddToHelpList();
+
+protected:
+	HelpUtil::Detail help_detail {};
+>>>>>>> f97ee951b (Switch shell cmd's to help utils, implement cmd categories)
```

Comment:
Should probably patch up this commit so it can be compiled? If I'm correct the idea is that no commit should fail compilation.

Reply:
Oops?